skimage/transform/hough_transform.py

```diff
@@ -17,15 +17,15 @@ def hough_line(img, theta=None):
         Input image with nonzero values representing edges.
     theta : 1D ndarray of double
         Angles at which to compute the transform, in radians.
-        Defaults to -pi/2 .. pi/2
+        Defaults to -pi/2 .. pi/2 with 180 values.
 
     Returns
     -------
-    H : 2-D ndarray of uint64
+    hspace : 2-D ndarray of uint64
```

Comment:
@JDWarner We are not on the same frequency :)

I changed the name of `hough_line` output to match the name used for `hough_line_peak` input. Ideally, I would have preferred to name everything 'distances' rather than 'dists'. If I do so, I need to change the API of `hough_line_peak`.

Hope my explanation is better ;)